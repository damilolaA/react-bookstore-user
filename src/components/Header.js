import React from 'react';

const Header = () => {

	return (
		<div>
			<div className="top-bar">
		    <div className="top-nav">
		      <a href="index.html"><h3 className="brand"><span>B</span>rain<span>F</span>ood</h3></a>
		      <ul className="top-nav-list">
		        <li className="top-nav-listItem Home"><a href="index.html">Home</a></li>
		        <li className="top-nav-listItem catalogue"><a href="catalogue.html">Catalogue</a></li>
		        <li className="top-nav-listItem login"><a href="login.html">Login</a></li>
		        <li className="top-nav-listItem register"><a href="registration.html">Register</a></li>
		        <li className="top-nav-listItem cart">
		          <div className="cart-item-indicator">
		            <p>12</p>
		          </div>
		          <a href="cart.html">Cart</a>
		        </li>
		      </ul>
		      <form className="search-brainfood">
		        <input type="text" className="text-field" placeholder="Search all books"/>
		      </form>
		    </div>
		  </div>
		</div>
	);
}

export default Header;