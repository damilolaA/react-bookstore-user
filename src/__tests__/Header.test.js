import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import Header from '../components/Header';

Enzyme.configure({ adapter: new Adapter() });

describe('Header Component', () => {
	it('should match header snapshot', () => {
		const wrapper = shallow( <Header/> );

		expect(toJson(wrapper)).toMatchSnapshot();
	});
});