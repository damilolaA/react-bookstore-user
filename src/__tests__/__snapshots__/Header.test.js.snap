// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should match header snapshot 1`] = `
<div>
  <div
    className="top-bar"
  >
    <div
      className="top-nav"
    >
      <a
        href="index.html"
      >
        <h3
          className="brand"
        >
          <span>
            B
          </span>
          rain
          <span>
            F
          </span>
          ood
        </h3>
      </a>
      <ul
        className="top-nav-list"
      >
        <li
          className="top-nav-listItem Home"
        >
          <Link
            replace={false}
            to="/"
          >
            Home
          </Link>
        </li>
        <li
          className="top-nav-listItem catalogue"
        >
          <Link
            replace={false}
            to="/catalogue"
          >
            Catalogue
          </Link>
        </li>
        <li
          className="top-nav-listItem login"
        >
          <Link
            replace={false}
            to="/login"
          >
            Login
          </Link>
        </li>
        <li
          className="top-nav-listItem register"
        >
          <Link
            replace={false}
            to="/register"
          >
            Register
          </Link>
        </li>
        <li
          className="top-nav-listItem cart"
        >
          <div
            className="cart-item-indicator"
          >
            <p />
          </div>
          <Link
            replace={false}
            to="/cart"
          >
            Cart
          </Link>
        </li>
      </ul>
      <form
        className="search-brainfood"
      >
        <input
          className="text-field"
          placeholder="Search all books"
          type="text"
        />
      </form>
    </div>
  </div>
</div>
`;
